src/sentry/api/helpers/group_index/delete.py

```diff
@@ -110,6 +105,33 @@ def delete_group_list(
         )
 
 
+def delete_group_hashes(
+    project_id: int,
+    group_ids: Sequence[int],
+    seer_deletion: bool = False,
+) -> None:
+    hashes_batch_size = options.get("deletions.group-hashes-batch-size")
+    total_hashes = GroupHash.objects.filter(project_id=project_id, group_id__in=group_ids).count()
+
+    for i in range(0, total_hashes, hashes_batch_size):
+        qs = GroupHash.objects.filter(project_id=project_id, group_id__in=group_ids)[
+            i : i + hashes_batch_size
+        ]
```

Comment:
You don't need to increase `i` here. The rows will be deleted after each iteration, so you just need `[:hashes_batch_size]`